app/controllers/open_ai_controller.rb

```diff
@@ -0,0 +1,50 @@
+class OpenAiController < ApplicationController
+  def show
```

Comment:
To tamp down on abuse, maybe this could be login-gated and/or rate-limited.

Reply:
For sure, for now I will see how costly it is. The wiki isn't used _that_ much, so it might be fine. Currently the api key is rate limited to 5 euro a month, we'll see how long that lasts